### ISSUE TITLE
Paginate results

### ISSUE THREAD

**Issue by babacarcissedia** [NgallaJuke/Glance-Backend#12]
# Scope
Right now user controller is returning all the records. This can be an issue when we reach million of record or even less.

# Acceptance
As a user, I can see records (ie user, post, etc) paginated

**Comment by NgallaJuke** [comment]
Yes ! I was also aware of this. Since the App was never meant for prod in the beginning, I just look over that part. But I think Adding it is good idea since I'm whiling now to deploy it at some point.
